env0/resource_environment.go

```diff
@@ -232,9 +249,18 @@ func setEnvironmentConfigurationSchema(d *schema.ResourceData, configurationVari
 		if configurationVariable.Description != "" {
 			variable["description"] = configurationVariable.Description
 		}
+		if configurationVariable.Regex != "" {
```

Comment:
envrionment is a bit of a mess...
The only that does not use readResource and writeResource.

Reply:
> The only that does not use readResource and writeResource.

because?